package.json

```diff
@@ -1,163 +1,96 @@
-    "aurelia-http-client": "^1.3.1",
-    "aurelia-loader-nodejs": "^1.1.0",
-    "aurelia-pal-nodejs": "3.0.0-rc.1",
-    "aurelia-polyfills": "latest",
-    "aurelia-router": "^1.7.1",
-    "aurelia-testing": "^1.1.0",
-    "aurelia-webpack-plugin": "^5.0.6",
+    "@aurelia/testing": "latest",
+    "@aurelia/ts-jest": "latest",
+    "@aurelia/webpack-loader": "latest",
+    "@types/jest": "^29.5.2",
+    "@types/node": "^18.11.18",
+    "@typescript-eslint/eslint-plugin": "^5.60.1",
+    "@typescript-eslint/parser": "^5.60.1",
+    "autoprefixer": "^10.4.14",
     "bootstrap": "^5.3.1",
-    "clean-webpack-plugin": "^4.0.0",
-    "copy-webpack-plugin": "^11.0.0",
-    "copyfiles": "^2.4.1",
-    "css-loader": "6.8.1",
-    "eslint": "^8.46.0",
-    "eslint-plugin-import": "^2.28.0",
-    "eslint-plugin-prefer-arrow": "^1.2.3",
-    "fetch-jsonp": "^1.3.0",
-    "file-loader": "6.2.0",
+    "css-loader": "^6.8.1",
+    "dotenv-webpack": "^8.0.1",
+    "eslint": "^8.43.0",
     "font-awesome": "^4.7.0",
-    "html-loader": "4.2.0",
-    "html-webpack-plugin": "5.5.3",
-    "i18next-xhr-backend": "^3.2.2",
-    "isomorphic-fetch": "^3.0.0",
-    "jest": "^29.6.2",
-    "jest-cli": "^29.6.2",
-    "jest-environment-jsdom": "^29.6.2",
-    "jest-extended": "^4.0.1",
-    "jsdom-global": "^3.0.2",
-    "mini-css-extract-plugin": "^2.7.6",
+    "html-webpack-plugin": "^5.5.3",
+    "htmlhint": "^1.1.4",
+    "jest": "^29.5.0",
+    "jest-environment-jsdom": "^29.5.0",
+    "jest-transform-stub": "^2.0.0",
     "moment-mini": "^2.29.4",
-    "npm-run-all2": "^6.0.6",
-    "promise-polyfill": "^8.3.0",
-    "release-it": "^16.1.3",
+    "postcss": "^8.4.24",
+    "postcss-loader": "^7.3.3",
     "rimraf": "^5.0.1",
-    "rxjs": "^7.8.1",
-    "sass": "^1.64.2",
-    "sass-loader": "^13.3.2",
-    "servor": "^4.0.2",
-    "style-loader": "3.3.3",
-    "ts-jest": "^29.1.1",
+    "style-loader": "^3.3.3",
     "ts-loader": "^9.4.4",
-    "ts-node": "^10.9.1",
-    "typescript": "^4.9.5",
-    "webpack": "^5.88.2",
+    "tslib": "^2.6.0",
+    "typescript": "^5.1.6",
+    "webpack": "^5.88.1",
+    "webpack-bundle-analyzer": "^4.9.0",
     "webpack-cli": "^5.1.4",
-    "webpack-dev-server": "^4.15.1"
-  },
-  "engines": {
-    "node": ">=16.15.0",
-    "npm": ">=8.5.0"
+    "webpack-dev-server": "^4.15.1",
+    "webpack-node-externals": "^3.0.0"
   },
-  "resolutions": {
-    "caniuse-lite": "1.0.30001519",
-    "semver": "^7.5.4"
+  "scripts": {
+    "lint:js": "eslint src test --ext .js,.ts",
+    "lint:html": "htmlhint -c .htmlhintrc src",
+    "lint": "npm run lint:js && npm run lint:html",
+    "pretest": "npm run lint",
+    "start": "webpack serve",
+    "build": "rimraf dist && webpack --env production",
+    "analyze": "rimraf dist && webpack --env production --analyze",
+    "test": "jest",
+    "prepare": "npm run build"
   },
-  "aurelia": {
-    "build": {
-      "resources": [
-        "aurelia-slickgrid/aurelia-slickgrid"
+  "jest": {
```

Comment:
could you put back the original stuff on top of the file (npm scripts, repo info and others...) and also this `jest` section is in a separate folder of its own so not required to be in here since that will fail my own unit tests.